packages/kit/src/views/Staking/components/ApproveBaseStake/index.tsx

```
@@ -418,30 +424,35 @@ export function ApproveBaseStake({
+
+        showEstimateGasAlert({
+          daysConsumed: formatStakingDistanceToNowStrict(
+            response.coverFeeSeconds,
+          ),
+          estFiatValue: response.feeFiatValue,
+          onConfirm: async (dialogInstance: IDialogInstance) => {
+            await dialogInstance.close();
+            await onNext();
+          },
+        });
+      }
```

ESLint

[error] Unnecessary 'else' after 'return'.

(no-else-return)